### PR TITLE
[Documentation] Update README.md with Accessibility testing results

### PR DESCRIPTION
**Pull request by lpeixin**
## Purpose
Wrap up a summary for accessibility testing on previous Android API devices.

## Does this introduce a breaking change?
```
[ ] Yes
[X] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[X] Documentation content changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,24 @@ callComposite.launch(context, options);
 
 For more details on Mobile UI Library functionalities visit the [API Reference Documentation](docs/api/CallComposite/Reference.md).
 
+### Accessibility
+
+Privious Android API devices could perform accessibility differently comparing to the latest version. We run through accessibility testing on previous Android API (21, 24, 27, 28) devices to detect the possible differences on accessibility performance.
```
Comment (Mrayyan) [inline README.md:100]: Run through -> ran though

**Review by Mrayyan — CHANGES_REQUESTED** [review]
Some spelling mistakes

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (lpx1989) [reply README.md]: addressed.

Comment (lpx1989) [reply README.md]: thx, addressed.

File: README.md [hunk 2]
```diff
@@ -95,6 +95,24 @@ callComposite.launch(context, options);
...
+
+#### [API 21](#tab/API21)
+```API 21 
+When focusing on buttons, screen reader will not announce "double tap to activate".
+There is no initial focus on setup screen.
+The state/selected change for video/mic buttons swich camera button and audio device select menu may not be announced.
```
[the remark this answers is not in the document]
Comment (lpx1989) [reply README.md]: thx for the suggestion. updated.

File: README.md [hunk 3]
```diff
@@ -95,6 +95,24 @@ callComposite.launch(context, options);
...
+The snackbar on setup screen with error message will not be focused and announced.
+```
+
+#### [API 27/28](#tab/API27_28)
+``` API 27/28
+The state/selected change for video/mic buttons swich camera button and audio device select menu may not be announced.
```
[the remark this answers is not in the document]
Comment (lpx1989) [reply README.md]: updated as well

**Review by Mrayyan — APPROVED** [review]

**Review by Mohtasim — APPROVED** [review]

**Pull request merged**